plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -474,8 +474,6 @@ static void tokudb_backup_before_stop_capt_fun(void *arg) {
       return;
     }
   }
-  // TODO : Backup lock not ported to 8.0
-  // (void)lock_binlog_for_backup(thd);
```

Comment:
This isn't exactly correct, we need to properly implement this in 8.0 for toku hot backup to actually work properly w.r.t. capturing binlog position.

Reply:
I will make the comment mention pfs.log_status instead

Reply:
LGTM